### PR TITLE
[TASK] Add .gitattributes

### PR DESCRIPTION
**Pull request by mbrodala**
This removes unnecessary resources from dist archives.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.php_cs.dist export-ignore
+/.phpstan.neon export-ignore
+/.styleci.yml export-ignore
+/.travis.yml export-ignore
+/ChangeLog export-ignore
```
Comment (IchHabRecht) [inline .gitattributes]: Please exclude the ChangeLog as I want it to be shipped in any case. Thank you!

Comment (mbrodala) [reply .gitattributes]: Done, wasn't sure about this.

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.php_cs.dist export-ignore
+/.phpstan.neon export-ignore
+/.styleci.yml export-ignore
+/.travis.yml export-ignore
+/Tests export-ignore
```
Comment (IchHabRecht) [inline .gitattributes]: Sorry to come back here once more, but according to the documentation, the path should end with an slash (https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes)

Comment (IchHabRecht) [reply .gitattributes]: Obviously I was able to change your pull request on my own ^^ Will merge as soon as travis is passed

Comment (mbrodala) [reply .gitattributes]: This is only essential if there could be a toplevel file called `Tests` which is obviously not the case here. ;-)

**Pull request merged**